### PR TITLE
HIP-573 Custom fees blanket exemptions API

### LINKED ISSUE #4509: HIP-573 Custom fees blanket exemptions API
### Problem

[HIP-573](https://hips.hedera.com/hip/hip-573) is adding blanket exemptions for custom fee collectors. The mirror node should support the new functionality.

### Solution

Consult the [feature checklist](https://github.com/hashgraph/hedera-mirror-node/blob/main/docs/checklist/feature.md) for a full list of potential tasks. The relevant ones should be:
- [ ] `openapi.yml`
- [ ] Model
- [ ] View model
- [ ] Controller
- [ ] Service
- [ ] Tests for each of the above
- [ ] `integrationDomainOps`
- [ ] Spec tests
- [ ] Acceptance REST model
- [ ] Acceptance client
- [ ] Acceptance feature

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:
[HIP-573 Custom fees blanket exemptions API](https://github.com/hashgraph/hedera-mirror-node/issues/4509)

This PR modifies the following:
* Add all_collectors_are_exempt to the openapi.yaml
* Change respective tests

**Related issue(s)**:

Fixes #4509

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rest/viewmodel/customFeeViewModel.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ class CustomFeeViewModel {
     }
 
     this.collector_account_id = EntityId.parse(customFee.collectorAccountId, {isNullable: true}).toString();
+    this.all_collectors_are_exempt = customFee.all_collectors_are_exempt;
```
Comment (steven-sheehy) [inline hedera-mirror-rest/viewmodel/customFeeViewModel.js:64]: nit: Move before amount so it's sorted in the response.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Please also update the acceptance tests (as noted in the feature checklist) to populate and verify in response of MirrorCustomFees

**Comment by mgoelswirlds** [comment]
> Details

Will add them.

File: hedera-mirror-rest/api/v1/openapi.yml [hunk 2]
```diff
@@ -2147,6 +2147,9 @@ components:
     FixedFee:
       type: object
       properties:
+        all_collectors_are_exempt:
+          type: boolean
+          example: false
```
Comment (xin-hedera) [inline hedera-mirror-rest/api/v1/openapi.yml:2152]: add `nullable: false`

**Review by xin-hedera — CHANGES_REQUESTED** [review]

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**